src/main/java/meteordevelopment/meteorclient/systems/modules/misc/ServerSpoof.java

```diff
@@ -60,31 +89,46 @@ private void onPacketSend(PacketEvent.Send event) {
             CustomPayloadC2SPacketAccessor packet = (CustomPayloadC2SPacketAccessor) event.packet;
             Identifier id = packet.getChannel();
 
-            if (id.equals(CustomPayloadC2SPacket.BRAND)) {
+            if (spoofBrand.get() && id.equals(CustomPayloadC2SPacket.BRAND))
                 packet.setData(new PacketByteBuf(Unpooled.buffer()).writeString(brand.get()));
-            }
-            else if (StringUtils.containsIgnoreCase(packet.getData().toString(StandardCharsets.UTF_8), "fabric") && brand.get().equalsIgnoreCase("fabric")) {
-                event.cancel();
+
+            if (channels.get()) for (String channel : channelsOut.get()) {
```

Comment:
Please don't use this scuffed formatting, put the for loop inside {} on a new line.